### DISCUSSION TITLE
Linking avatars in a scene to placeholder objects

### DISCUSSION THREAD

**Discussion by BikaSibila** [mozilla/hubs#3701] [Avatar Questions] [answered]
Hi, I would like to link a custom avatar to an object so that people can click on it and become that avatar. Here an example of a scene where this was done nicely. 
https://hubs.mozilla.com/tPmdd5H/area-for-virtual-art/

**Comment by j-conrad** [comment, marked as answer]
This is already possible and was done this way:

1. Click on the **'Place'** menu
2. Choose **'Avatar'**
3. Select an avatar
4. Your selected avatar will appear in the room as a clickable 2d image. Hovering over the image will reveal a **'Use Avatar'** button.

_NOTE: If you want that clickable image to remain when you return, you must use the **'Pin'** feature. Hold Spacebar to reveal the **'Pin'** button on the image._

https://gyazo.com/71e8df28faf9beffb40fda466bf30ab4

https://gyazo.com/f41c6639a6a92eb9042618a4b7336a7c
